File: testdata/multi_language_examples/csharp/Calculator.cs
using System;

namespace MyApp.Calculators
{
    public class Calculator
    {
        public int Add(int a, int b)
        {
            if (a > 0 && b > 0)
            {
                return a + b;
            }
            return 0;
        }

        public int Subtract(int a, int b)
        {
            return a - b;
        }
    }
}

File: testdata/multi_language_examples/cpp/calculator.cpp
#include <iostream>
#include "calculator.h"

int Calculator::add(int a, int b) {
    if (a > 0 && b > 0) {
        return a + b;
    }
    return 0;
}

int Calculator::subtract(int a, int b) {
    return a - b;
}
